packages/babel-plugin-i18n-calypso/src/index.js

```diff
@@ -56,6 +56,31 @@ const DEFAULT_HEADERS = {
  */
 const DEFAULT_DIR = 'build/';
 
+/**
+ * List of translate function to be handled for string extraction.
+ *
+ * @type {string[]}
+ */
+const TRANSLATE_FUNCTION_NAMES = [ '__', '_n', '_nx', '_x', 'translate' ];
+
+/**
+ * Set of keys which are valid to be assigned into a translation object.
+ *
+ * @type {string[]}
+ */
+const VALID_TRANSLATION_KEYS = [ 'msgid_plural', 'msgctxt' ];
+
+/**
+ * The order of arguments in @wordpress/i18n translate function.
+ *
+ * @type {object}
+ */
+const DEFAULT_FUNCTIONS_ARGUMENTS_ORDER = {
+	_n: [ 'msgid_plural' ],
```

Comment:
Can we add `translate` here?

Reply:
Added translate and did a small refactoring in https://github.com/Automattic/wp-calypso/pull/39280/commits/e13f5e830009b87f907a80174f79954c97e3056e to make it a bit more cleaner.